### ISSUE TITLE
logits must be 2-dimensional

### ISSUE THREAD

**Issue by alexhanna** [tensorflow/tensor2tensor#1512]
### Description

I'm trying to perform inference with a tensor2tensor model which I trained and am hosting on Cloud ML Engine. The model takes a text sequence and is supposed to return the next line of "poetry." It's based off [this lab](https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive/09_sequence/poetry.ipynb).

I am attempting to use the tensor2tensor serving API to accept a string and perform inference within a notebook. However, I keep getting this error:

     Error during model execution: AbortionError(code=StatusCode.INVALID_ARGUMENT, details="logits must be 2-dimensional
    [[Node: transformer/body/parallel_0/body/encoder/layer_0/self_attention/multihead_attention/dot_product_attention/attention_weights = Softmax[T=DT_FLOAT, _output_shapes=[[?,4,?,?]], _device="/job:localhost/replica:0/task:0/device:CPU:0"](transformer/body/parallel_0/body/encoder/layer_0/self_attention/multihead_attention/dot_product_attention/add)]]")

...

### Environment information

```
OS: Colab

$ pip freeze | grep tensor
mesh-tensorflow==0.0.5
tensor2tensor==1.11.0
tensorboard==1.13.1
tensorboardcolab==0.0.22
tensorflow==1.13.1
tensorflow-estimator==1.13.0
tensorflow-hub==0.3.0
tensorflow-metadata==0.13.0
tensorflow-probability==0.6.0
tensorflow-serving-api==1.13.0

$ python -V
Python 2.7.15rc1
```

**Comment by alexhanna** [comment]
Closing this out after training a new model and having more success.

**Comment by Cumberbatch08** [comment]
hello, did you have solution?
